Pages/footer.js

```diff
@@ -0,0 +1,27 @@
+const Page = require("./page");
+var  data = require("../Data/data.js");
+
+class Footer extends Page.Page{
+    constructor(){
+        super();
+        this.title="Footer";
+    }
+
+    // ACTIONS
+
+    validateTitleParagraph(titleParagraph){
```

Comment:
Can you try to combine these three methods into one, only difference could be for "data.x" part ?
You can add additional parameter and if/else or switch logic inside.

Reply:
I used switch logic for resolving this code duplication.